src/main/java/ch/wisv/areafiftylan/exception/InvalidTokenException.java

```diff
@@ -0,0 +1,15 @@
+package ch.wisv.areafiftylan.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+/**
+ * Created by beer on 11-3-16.
+ */
+
+@ResponseStatus(HttpStatus.UNAUTHORIZED)
```

Comment:
I'd turn this in something else then UNAUTHORIZED, because that's not the case. Maybe BAD_REQUEST or NOT_MODIFIED.